docs/java/testing-and-debugging.md

```diff
@@ -12,6 +14,31 @@ easily test long-running Workflows in seconds, without having to change your Wor
 You can use the provided `TestWorkflowEnvironment` with a Java unit testing framework of your choice,
 such as JUnit.
 
+## Setup testing dependency
+In order to start using the Java SDK test framework you need to add [`io.temporal:temporal-testing`](https://search.maven.org/artifact/io.temporal/temporal-testing) 
```

Comment:
- Recommend changing "In order to" to simply "To", per the "in order to" entry in [Word List](https://developers.google.com/style/word-list#letter-i) in the Google dev doc style guide.
- Suggest inserting a comma in "framework you" for readability.

```suggestion
To start using the Java SDK test framework, you need to add [`io.temporal:temporal-testing`](https://search.maven.org/artifact/io.temporal/temporal-testing) 
```